src/pages/directory.tsx

```diff
@@ -0,0 +1,39 @@
+import { api } from "~/utils/api";
+import { Button, ButtonGroup, Stack } from "@mui/material";
+import { useState } from "react";
+import Profile from "../components/directory/Profile";
+
+const Directory = () => {
+  const [current, setCurrent] = useState<string>("mentee");
```

Comment:
We can use[ Typescript enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) to our advantage here. You can make the enum names all caps and make the strings the way you want to display it. You can use toUpperCase for the argument to useQuery.

https://stackoverflow.com/a/74326516